src/data/faq.json

```diff
@@ -44,6 +44,17 @@
                                     "FAQ articles on general questions can be found <a href='#ramp_down_general' target='_blank'>here</a>, FAQ articles on implications regarding the app functions <a href='#ramp_down_effects' target='_blank'>here</a> and FAQ articles on alternatives <a href='#ramp_down_alternatives' target='_blank'>here</a>."
                                 ]
                             },
+                            {
+                                "title": "What can I do now? What do I need to know?",
+                                "anchor": "ramp_down_info",
+                                "active": false,
+                                "textblock": [
+                                    "At the end of the Corona-Warn-App development and the cutback of the functions you can still backup your entries and certificates (see also the advice in the <a href='#ramp_down_backup' target='_blank'>FAQ on backup of data</a>).",
+                                    "The exported data falls outside the scope of CWA and is therefore the responsibility of the user.  This data must be carefully handled by the user and manually deleted if deemed necessary or requested.",
+                                    "After April 30, 2023, the Corona-Warn-App is going to trigger the technical instruction to stop the ENF (exposure notification framework). This command will cause the smartphone’s operating system to stop exchanging random IDs and to delete all encounter information. ",
+                                    "The ENF can also be deactivated manually; you guidance on this can be found in the <a href='#ramp_down_exchange' target='_blank'>FAQ on random IDs</a>."

```

Comment:
"you guidance on this" =>
"your guidance on this"

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/3457/commits/808583def68f477dc609aa0b383454e8fd12dcc2, thanks!